types/page/index.d.ts

```diff
@@ -167,6 +169,11 @@ declare namespace PageJS {
          * Equivalent to page.exit('*', callback).
          */
         exit(callback: Callback): void;
+
+        /**
+         * This is the click handler used by page to handle routing when a user clicks an anchor like `<a href="/user/profile">`
+         */
+        clickHandler(e: UIEvent): void;
```

Comment:
should probably be a `MouseEvent` since it exists to handle click events

Reply:
Yes, I've been wondering, but opted more general one, would go 'MouseEvent' way though, thx!

Reply:
Done, thx!